text/0000-pack-buildpack-create.md

```diff
@@ -0,0 +1,74 @@
+# Meta
+[meta]: #meta
+- Name: Pack Command to Create a Buildpack Repo
+- Start Date: 2021-01-19
+- Author(s): [jkutner](https://github.com/jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a new command in Pack that would create the scaffolding for a new buildpack.
+
+# Definitions
+[definitions]: #definitions
+
+- *Scaffold* - the essential files and directory required for a buildpack
+
+# Motivation
+[motivation]: #motivation
+
+Every new buildpack requires a few essential files and directories. We can make buildpack development easier by automatically generating these with a single command.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Add a new command to Pack:
+
+```
+$ pack buildpack create <id> \
+    --language <lang> \
+    --stack <stack>
```

Comment:
I'd like users to be able to provide multiple stacks

Reply:
yea, that was my intention here (like the `--buildpack` flag). I'll call that out.